AppServicesUsageSamples/demo/src/main/java/io/realm/appservicesusagesamples/propertyencryption/models/CustomData.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2023 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.realm.appservicesusagesamples.propertyencryption.models
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+/**
+ * App services user custom data. It contains the resources required to achieve property level
+ * encryption:
+ * 1. Encryption key, stored and password-protected in [keyStore].
+ * 2. Field encryption cipher spec, defined in [PLECipherSpec],
+ */
+@Serializable
+class CustomData(
```

Comment:
Should we change the name so it is explicit that this a specific interpretation of the user's custom data specific to this app. Maybe `CustomKeyConfiguration` ... #naming

Reply:
`PropertyEncryptionConfiguration`